NopApp.Core/NopApp.Models/DbModels/Subscription.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NopApp.Models.DbModels
+{
+    public class Subscription
+    {
+        public string Id { get; set; }
+        public User User { get; set; }
+        public Offer Offer { get; set; }
+        public DateTime StarDate { get; set; }
+        public DateTime EndDate { get; set; }
+    }
+}
```

Comment:
We should store the total price of the subscription when creating it (when the payment happens). The price of the offer might change later, but the paid amount should be stored